src/main/Config.cpp

```diff
@@ -517,6 +517,28 @@ Config::load(std::string const& filename)
     }
 }
 
+void
+Config::addSelfToValidators(
+    std::vector<ValidatorEntry>& validators,
+    std::unordered_map<std::string, ValidatorQuality> const& domainQualityMap)
+{
+    auto it = domainQualityMap.find(NODE_HOME_DOMAIN);
+    ValidatorEntry self;
+    self.mKey = NODE_SEED.getPublicKey();
+    self.mHomeDomain = NODE_HOME_DOMAIN;
+    self.mName = "self";
+    self.mHasHistory = false;
+    if (it != domainQualityMap.end())
+    {
+        self.mQuality = it->second;
+    }
+    else
+    {
+        LOG(WARNING) << "Defaulting self to LOW priority";
+        self.mQuality = ValidatorQuality::VALIDATOR_LOW_QUALITY;
```

Comment:
The choice of making this `VALIDATOR_LOW_QUALITY` seems arbitrary, so a comment is warranted here.

Reply:
errors out now